docker/package/scripts/tezos-baking-prestart

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+set -euo pipefail
+
+tezos_client="/usr/bin/tezos-client"
+
+config="$DATA_DIR/config"
+
+mkdir -p "$DATA_DIR"
+
+if [[ ! -f $config ]]; then
+    "$tezos_client" --base-dir "$DATA_DIR" \
+                    --endpoint "$NODE_RPC_ENDPOINT" \
+                    config init
+else
+    "$tezos_client" --base-dir "$DATA_DIR" \
+                    --endpoint "$NODE_RPC_ENDPOINT" \
+                    config update
```

Comment:
the two commands appear to differ only in init vs update, perhaps it makes sense to spell it out in the way that makes this apparent, something like this:

```
"$tezos_client" --base-dir "$DATA_DIR" \
                    --endpoint "$NODE_RPC_ENDPOINT" \
                    config $([[ -f $config ]] && echo "update" || echo "init")
```